js/sound.js

```diff
@@ -0,0 +1,48 @@
+const backgroundAudio = ["audio/music/epic1.mp3", "audio/music/relaxing1.mp3"];
+var backgroundNum = Math.floor(Math.random() * backgroundAudio.length);
+var backgroundPlay = new Audio(backgroundAudio[backgroundNum]);
+var audioPlay = null;
+$( document ).ready(function() {
+    startBackgroundAudio();
+});
+function startBackgroundAudio() {
+    console.log(backgroundPlay);
+    backgroundPlay.play();
+}
+function nextBackgroundAudio() {
+    backgroundPlay.pause();
+    if (backgroundNum !== backgroundAudio.length - 1) {
+        backgroundNum = backgroundNum + 1
+    }
+    backgroundPlay = new Audio(backgroundAudio[backgroundNum]);
+    console.log(backgroundPlay);
+    backgroundPlay.play();
+}
+function previousBackgroundAudio() {
+    backgroundPlay.pause();
+    if (backgroundNum !== 0) {
+        backgroundNum = backgroundNum - 1
+    }
+    backgroundPlay = new Audio(backgroundAudio[backgroundNum]);
+    console.log(backgroundPlay);
+    backgroundPlay.play();
+}
+function stopBackgroundAudio() {
+    backgroundPlay.pause();
+}
+function setVolume() {
+    if (audioPlay !== null) {
+        audioPlay.volume = document.getElementById("volume1").value;
+    }
+    backgroundPlay.volume = document.getElementById("volume1").value;
+}
+function stopAudio() {
+    if (audioPlay !== null) {
+        audioPlay.pause();
+    }
+}
+function playAudio (audio) {
+    audioPlay = new Audio("audio/" + audio);
+    console.log(audioPlay);
```

Comment:
We also need to show the legalNotice on the front page: `"Now playing: " + legalNotice`

Reply:
@niccokunzmann I can add that in another PR. It would just make my work a bit more convenient. If that is okay with you, can we please merge?

Reply:
Erm. Yes, if we do not play the music. Is that ok?

Reply:
I mean: if you promise and it takes a long time, all this time, people who view the site do not have the credit running for bensound which means, we violate their license. This is not really appreciation for their work in my eyes.

Reply:
What do you think?

Reply:
Done! Making the PR now. The new legal declaration way is awesome!